pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -442,6 +442,8 @@ func validateAndGetNetworkPluginName(originClient *osclient.Client, pluginName s
 		clusterNetwork, err := originClient.ClusterNetwork().Get(sdnapi.ClusterNetworkDefault)
 		if err == nil {
 			return clusterNetwork.PluginName, nil
+		} else if !kerrs.IsNotFound(err) {
```

Comment:
Against an old master or one that has not reconciled policy since upgrade yet, will this return a forbidden error?